com.zsmartsystems.zigbee.serial/src/main/java/com/zsmartsystems/zigbee/serial/ZigBeeSerialPort.java

```diff
@@ -273,19 +273,21 @@ public void purgeRxBuffer() {
         }
     }
 
-    public boolean setDtr(boolean state) {
+    @Override
```

Comment:
These methods should not be changed. They can't be defined in the interface as per the other comment, so th `@Override` should be removed, and my preference would be not to change the return statement as I don't know if this is used by other users. If it is, it will be a breaking change that I would prefer to avoid.